lib/subroutine/op.rb

```diff
@@ -48,6 +48,18 @@ def initialize(inputs = {})
       yield self if block_given?
     end
 
+    def inspect
+      values = provided_params.to_a
+      values.map! do |(key, value)|
+        "#{key}: #{value.inspect}"
+      end
+      values.sort!
+      values = values.join(", ")
+
+      oid = format('%x', (object_id << 1))
```

Comment:
Can we add a comment or something to explain the why behind this? Unclear to me why we're shifting the object ID.